ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -233,27 +232,99 @@ const LocalStandardDisks = ({ idPrefix, setIsFormValid, onAddErrorNotification }
+              id="local-disks-bulk-select-none"
+            >
+                {_("Select none")}
+            </DropdownItem>,
+            <DropdownItem
+              key="select-all"
+              component="button"
+              aria-label={_("Select all disks")}
+              onClick={() => setSelectionForAllDisks(true)}
+              id="local-disks-bulk-select-all"
+            >
+                {_("Select all")}
+            </DropdownItem>,
+        ];
+
+        return (
+            <Dropdown
+              onSelect={onSelect}
+              toggle={
+                  <DropdownToggle
+                    splitButtonItems={[
+                        <DropdownToggleCheckbox
+                          key="select-multiple-split-checkbox"
+                          aria-label={_("Select multiple disks")}
+                          isChecked={selectedDisksCnt > 0}
+                        >
+                            {selectedDisksCnt > 0 ? cockpit.format(_("$0 selected"), selectedDisksCnt) : ""}
+                        </DropdownToggleCheckbox>
+                    ]}
+                    onToggle={onToggle}
+                    id="local-disks-bulk-select-toggle"
+                  />
+              }
+              isOpen={isOpen}
+              dropdownItems={dropdownItems}
+            />
+        );
+    };
+
     return (
         <Form>
-            <FormGroup
-              label={_("Local standard disks")}
-              labelIcon={
-                  <Flex display={{ default: "inlineFlex" }}>
+            <FormSection
+              title={
+                  <Flex spaceItems={{ default: "spaceItemsXs" }}>
+                      <FlexItem><h3>{_("Local standard disks")}</h3></FlexItem>
                       <FlexItem>{localDisksInfo}</FlexItem>
-                      <FlexItem>{diskSelectionLabel}</FlexItem>
                   </Flex>
               }
-              labelInfo={rescanDisksButton}
-              isRequired
             >
-                <ListingTable
-                  aria-labelledby="installation-destination-local-disk-title"
-                  {...(totalDisksCnt > 10 && { variant: "compact" })}
-                  columns={localDisksColumns}
-                  onSelect={(_, isSelected, diskId) => setDisks({ ...disks, [Object.keys(disks)[diskId]]: isSelected })}
-                  rows={localDisksRows}
-                />
-            </FormGroup>
+                <FormGroup>
+                    <Toolbar>
+                        <ToolbarContent>
+                            <ToolbarItem variant="bulk-select"><DropdownSplitButtonText /></ToolbarItem>
+                            <ToolbarItem variant="separator"><Divider orientation={{ default: "vertical" }} /></ToolbarItem>
+                            <ToolbarItem>{rescanDisksButton}</ToolbarItem>
+                        </ToolbarContent>
+                    </Toolbar>
+                    <ListingTable
```

Comment:
Maybe the toolbar and the table could be also defined separately? It is just that every reorganization of the elements requires to move all these blocks of code even when they are not changed at all.

Reply:
Fixed.